<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>WidgetToolSelect</class>
 <widget class="QWidget" name="WidgetToolSelect" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Selektion</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>2</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item row="1" column="0" colspan="2" >
       <widget class="QPushButton" name="button_SelectAll" >
        <property name="toolTip" >
         <string>Selektiert alle Objekte (STRG+A)</string>
        </property>
        <property name="statusTip" >
         <string>Selektiert alle Objekte (STRG+A)</string>
        </property>
        <property name="whatsThis" >
         <string>Selektiert alle Objekte (STRG+A)</string>
        </property>
        <property name="text" >
         <string>Alles Selektieren</string>
        </property>
        <property name="shortcut" >
         <string>Ctrl+A</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QPushButton" name="button_Delete" >
        <property name="toolTip" >
         <string>Löscht alle selektierten Objekte (Entfernen)</string>
        </property>
        <property name="statusTip" >
         <string>Löscht alle selektierten Objekte (Entfernen)</string>
        </property>
        <property name="whatsThis" >
         <string>Löscht alle selektierten Objekte</string>
        </property>
        <property name="text" >
         <string>Löschen</string>
        </property>
        <property name="shortcut" >
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Objekt-Name</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>5</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLineEdit" name="lineEdit_ObjectName" />
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="label_ObjectsCount" >
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Selektiert Objekte:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2" >
       <widget class="QGroupBox" name="groupBox_3" >
        <property name="title" >
         <string>Objekt-Beschreibung</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>5</number>
         </property>
         <item row="0" column="0" >
          <widget class="QTextEdit" name="textEdit_ObjectDesc" />
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
